### PR TITLE
LPD-39192 - getCountriesPage response does not match the localization selection of available languages configured on Instance Settings

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-39192

Original PR: https://github.com/liferay-commerce/liferay-portal/pull/5652

cc @smottal

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

**Review by smottal — CHANGES_REQUESTED** [review]

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/test/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/PostalAddressUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.headless.admin.user.internal.dto.v1_0.util;
```
[the remark this answers is not in the document]
Comment (brbalazs) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/test/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/PostalAddressUtilTest.java:6]: @smottal I put test at the very end then SF complained and then removed it. If you check OpenAPIUtilTest or the other 2 examples, you will see that they did not use test there either. Only top folder is called test.

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/test/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/PostalAddressUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+						HashMapBuilder.put(
+							"en_US", "United States"
+						).put(
+							"pt_BR", "Brasil"
+						).build(
+						).forEach(
+							titleMap::put
+						);
+
+						return titleMap;
```
[the remark this answers is not in the document]
Comment (brbalazs) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/test/java/com/liferay/headless/admin/user/internal/dto/v1_0/util/PostalAddressUtilTest.java:75]: @smottal you did not specify "where" to put it in your example, but I get your point, it should be:

```
Map<String, String> titleMap = HashMapBuilder.put(
    "en_US", "United States"
).put(
    "pt_BR", "Brasil"
).build();
```

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:relevant

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:relevant

**Comment by brbalazs** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**